examples/alarmBot/abilities.ts

```diff
@@ -57,22 +58,18 @@ export const abilities = [
             isValid: true
           }
         },
-        onFill: (value) => `ok! time is set to ${value}.`
+        onFill: (s, value) => `ok! time is set to ${value}.`
```

Comment:
I saw the feature request, do we want to hold on this PR and get the parameter order changed first? It should be a  low impact change and we can push out a new `alpha`?

Reply:
@dnastrain's PR addresses this feature request that was accepted and is now in `alpha.10`.

Reply:
Dan's PR addresses this fix.